lib/capybara_discoball.rb

```diff
@@ -1,4 +1,5 @@
 require 'capybara_discoball/version'
+require 'capybara_discoball/server'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.